libevmasm/Exceptions.h

```diff
@@ -28,6 +28,7 @@ namespace solidity::evmasm
 {
 
 struct AssemblyException: virtual util::Exception {};
+struct AssemblyImportException: virtual util::Exception {};
```

Comment:
Should this inherit from `AssemblyException` too?
```suggestion
struct AssemblyImportException: virtual AssemblyException {};
```